backend/src/routes/criteria.ts

```diff
@@ -0,0 +1,42 @@
+import { Router } from 'express';
+import Criteria from 'models/criteria';
+
+const router = Router();
+
+router.get('/', (req, res) => {
```

Comment:
From a style perspective how do you want to handle unused variables. I think the standard is to replace it with `_`